kitsune/sumo/static/sumo/scss/form-step.styles.scss

```diff
@@ -19,41 +19,60 @@
   color: var(--color-text-form-step);
 }
 
-.hidden {
-  visibility: hidden;
-}
-
-.error-tooltip:not(.hidden) + input:invalid,
-input:not(:focus):not(:placeholder-shown):invalid {
-  border: 2px solid p.$color-red-60;
-}
-
-.error-tooltip:not(.hidden) + input:focus:invalid {
-  box-shadow: 0 0 0 2px rgba(226, 40, 80, 0.5);
+.visible {
+  visibility: visible;
 }
```

Comment:
This should probably go after the .tooltip rule, because it's being overridden by the `visibility: hidden;` rule on line 38, even with the visible class.